contents/docs/integrate/ingest-historic-data.mdx

```diff
@@ -0,0 +1,143 @@
+---
+title: Ingest Historical Data
+sidebar: Docs
+showTitle: true
+---
+
+import { Tabs } from 'antd'
+export const TabPane = Tabs.TabPane
+
+Historical data ingestion (or importing data), opposed to [live data ingestion](/docs/integrate/ingest-live-data), is the process of transporting data from external sources into PostHog so you can benefit from PostHog product analytics on historical data. It may be that you have historical data that you want to analyze along with new live data or that you have a requirement to periodically import data from third-party sources to augment your live data.
+
+Whatever the reason for the historical data ingestion, this guide covers what to consider during that process.
+
+The three main factors to consider are:
+
+- [Data ingestion process](#data-ingestion-process): how to get the event data from the third-party source into PostHog
+- [Importing events](#importing-events): Sending the events captured in the third-party data source into PostHog as custom events
+- [User identification](#user-identification): How to identify users within PostHog and ties those users back to the user within original data source
+
+Historical data is sent to PostHog using either a [server library](/docs/integrate/overview#server-libraries) or the [PostHog API](/docs/api/overview). For more information see the [Importing events section](#importing-events).
+
+## Data ingestion process
+
+PostHog doesn't presently offer any specific tools to import or migrate historical data from third-party data sources. However, since the third-party data source is highly likely to offer an API you can use the power of software to import the data from one or more sources to PostHog.
```

Comment:
Suggest we make this intro paragraph more positive / confidence-inspiring - given this comes up a lot, we do have a good answer to it below. I wouldn't start with "we don't have X".

Reply:
Resolved in https://github.com/PostHog/posthog.com/pull/1680/commits/9bd24a5e4cf755a35d890533e2feb73c25695468